src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("/fonts/Avenir.ttc"),
+    url("/fonts/Avenir-Book.ttf"),
+    url("/fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.wrapper {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background: #fff;
+  border-radius: 4px;
+  font-weight: 900;
+  color: #3d4e61;
+  font-family: inherit;
+}
+
+.input::placeholder {
+  font-weight: 300;
+}
+
+.input__search {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input__search--small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+}
+
+.wrapper::before {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  background-image: url(/images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.wrapper__big::before {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.wrapper__small::before {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
```

Comment:
same stuff like with inputs, default wrapper and big/small.
--small is modifier, if you are using BEM

Reply:
I'm a newbie with the using BEM *(  
I'll change it. Thanks!